roles/foreman_installer/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: 'Run installer'
+  shell: >
```

Comment:
```suggestion
  command: >
```

There is no need for a `shell` here

Reply:
I wanted to protect the foreman-installer's back. (Friday dad jokes, you're welcome)